pages/Messages/Signatures.md

```diff
@@ -51,28 +53,28 @@ For example, given the following DSNP broadcast message:
 This would be the expected serialization:
 
 ```
-contentHash0x67890fromAddress0x12345urihttps://www.projectliberty.io/
+\x19Ethereum Signed Message:\n\x69contentHash0x67890fromAddress0x12345urihttps://www.projectliberty.io/
 ```
 
 This would be the expected hash:
 
 ```
-0x45c42591e0154088325055f26664002bf05f211db9e5d7c7bfc588dc309698e0
+0x4c233fc1d71f92e4b8b7408291d45b5aa2ec3689d14091abc9207c4eca3886ad
 ```
 
 And the generated signature would be a string unique to the signing user's keys but looking something like this:
 
 ```
-LJbwyo6oxKd2GRxsUFwxUN1YwBzE9UC4WPvScsh0+vALkocQn60QjgkNd9CB0JUKfVTQdOlTm5gzaengzgmhDw==
+0xf9d3010a7b1f199b5ea58473908c0c78a5cb04fdead2a05a7961c9a7ff87f1e81c654605b50f65b7d255177b0b49bbd97fa07fb88f9a7ea90565baa96b68e61e
 ```
 
 ## Verifying Messages
 
 Verifying announcements can be done by repeating the serialization and hashing steps from the signing process then validating the generated hash against the publishing user's public key.
 The user's public key can be fetched from the Identity contract as described in the [Identity spec](/Identity/Overview).
 
-Given the message and signature provided in the previous example, the following public key should return true when verifying:
+Given the message and signature provided in the previous example, the following public key should return a matching hash when verifying:
 
 ```
-+IUGiZsGHq+3s/Zgxl8TQMRgydvsOX1hUwMzHykoqGw=
+0x033dd49eb9790f8473399093c10eead398bbd76eae0905b9b26fd550aced63e858
```

Comment:
This should be a public key right?

Reply:
This is a public key. I just encoded it as hex instead of base64 for consistency.

Reply:
Fixed